Mage/src/main/java/mage/game/combat/Combat.java

```diff
@@ -983,7 +983,7 @@ public boolean checkBlockRestrictionsAfter(Player player, Player controller, Gam
                     for (Ability ability : entry.getValue()) {
                         if (!effect.canBlockCheckAfter(ability, game)) {
                             if (controller.isHuman()) {
-                                game.informPlayer(controller, blockingCreature.getLogName() + " can't block this way.");
+                                game.informPlayer(controller, new StringBuilder(blockingCreature.getLogName()).append(" can't block this way.").toString());
```

Comment:
It appears my local file for some reason undid changes from https://github.com/magefree/mage/commit/c0cf22bbf784ce726a82f59f520ee0bcd79daae3 I'm fixing it to redo the changes